py/makemoduledefs.py

```diff
@@ -59,18 +59,21 @@ def generate_module_table_header(modules):
     """
 
     # Print header file for all external modules.
-    mod_defs = []
+    mod_defs = set()
     print("// Automatically generated by makemoduledefs.py.\n")
     for module_name, obj_module, enabled_define in modules:
         mod_def = "MODULE_DEF_{}".format(module_name.upper())
-        mod_defs.append(mod_def)
+        mod_defs.add(mod_def)
         print(
             (
                 "#if ({enabled_define})\n"
                 "    extern const struct _mp_obj_module_t {obj_module};\n"
+                "    #undef {mod_def}\n"
```

Comment:
Does this mean that if two or more modules with the same name are enabled, then the last one (according to `sorted()`) "wins"?